src/wallets/EthersWallet.ts

```diff
@@ -89,34 +72,62 @@ export class EthersWallet implements TLWallet {
   }
 
   /**
-   * Create a serialized wallet following the current format from an encryptedKeystore
-   * Can be used to migrate from serialized wallet version 0 to serialized wallet version 1
-   * @param encryptedKeystore
+   * Create a serialized wallet following the current format from an encryptedKeystore.
+   * Can be used to migrate from serialized wallet version 0 to serialized wallet version 2.
+   * @param encryptedKeystore Standard serialized encrypted JSON keystore.
    */
   public serializeWallet(encryptedKeystore: string): string {
-    const deserializedWallet: EthersWalletSchema = {
+    const deserializedWallet: EncryptedTLWalletSchema = {
       TLWalletVersion: TL_WALLET_VERSION,
-      ethersKeystore: encryptedKeystore,
-      walletType: WALLET_TYPE_ETHERS
+      walletType: WALLET_TYPE_ETHERS,
+      meta: {
+        ethersKeystore: encryptedKeystore
+      }
     }
+    const serializedWallet = JSON.stringify(deserializedWallet)
+    return serializedWallet
+  }
 
-    const serializedWallet: string = JSON.stringify(deserializedWallet)
+  /**
+   * Encrypts and serializes the given wallet.
+   * @param ethersWallet `TLWallet` of type `WALLET_TYPE_ETHERS`.
+   * @param password Password to encrypt wallet with.
+   * @param progressCallback Optional encryption progress callback.
+   */
+  public async encryptWallet(
```

Comment:
I understood it that we made this function to return the standard format, so we should just return the encrypted keystore here.

Reply:
changed it so that it only returns the string as returned by `ethers.Wallet.encrypt`